test/extended/templates/templateservicebroker_security.go

```diff
@@ -100,32 +109,36 @@ var _ = g.Describe("[templates] templateservicebroker security test", func() {
 
 	provision := func(username string) error {
 		g.By("provisioning a service")
-		_, err := brokercli.Provision(context.Background(), instanceID, &api.ProvisionRequest{
+		_, err := brokercli.Provision(context.Background(), &user.DefaultInfo{Name: username}, instanceID, &api.ProvisionRequest{
```

Comment:
I see, so yeah along these same lines, you should just add the proper groups to the user you're passing, rather than granting the user new permissions.  Then we'd also be testing the group-based SAR checks.